syntax = "proto3";

//package grpcdemo;

// Interface exported by the server.
service DemoService {
   rpc GetUserProfile(UserProfileRequest) returns (UserProfileResponse) {}
}

// The request message containing the user's name.
message UserProfileRequest {
  int32 id = 1;
}

message UserProfileResponse {
  string last_name = 1;
  string first_name = 2;
  string phone = 3;
  string job_title = 4;
  bool active = 5;
  int32 id = 6;
}


